docs/architecture/glossary.rst

```diff
@@ -3,70 +3,131 @@ Glossary
 
 .. glossary::
 
-  basic element
   BEL
-  basic logic element
   BLE
-    For example a LUT5, LUT6, CARRY4, or MUX, but not PIPs.
+  Basic element (BEL) or basic logic element (BLE)
```

Comment:
Having these on separate lines defines multiple terms that all have the same definition (see http://www.sphinx-doc.org/en/stable/markup/para.html#glossary).  As is, this breaks any :term references on other pages.

Reply:
Thanks! Will do. I fixed the DCO check and put the BEL/BLE terms on separate lines. I haven't yet run the build to check errors - that's my next step.